ledger-tool/src/program.rs

```diff
@@ -131,18 +131,61 @@ fn load_blockstore(ledger_path: &Path, arg_matches: &ArgMatches<'_>) -> Arc<Bank
     bank
 }
 
-pub trait RunSubCommand {
-    fn run_subcommand(self) -> Self;
+pub trait ProgramSubCommand {
+    fn program_subcommand(self) -> Self;
 }
 
-impl RunSubCommand for App<'_, '_> {
-    fn run_subcommand(self) -> Self {
+impl ProgramSubCommand for App<'_, '_> {
+    fn program_subcommand(self) -> Self {
         self.subcommand(
+            SubCommand::with_name("program")
+        .about("Run to test, debug, and analyze on-chain programs.")
+        .subcommand(
+            SubCommand::with_name("cfg")
+                .about("generates Control Flow Graph of the program.")
+                .arg(
+                    Arg::with_name("PROGRAM")
+                        .help(
+                            "Program file to use. This is either an ELF shared-object file to be executed, \
+                             or an assembly file to be assembled and executed.",
+                        )
+                        .required(true)
+                        .index(1)
+                )
+                .arg(
+                    Arg::with_name("max_genesis_archive_unpacked_size")
```

Comment:
Same as `PROGRAM` arg

Reply:
Initially I did that, and then I thought it's better to have it in each subcommand because that way it appears in specific subcommand --help message.

Reply:
These my two comment replies are in wrong order. My previous response was to this comment, and this response is to your previous comment.

Reply:
> that doesn't work somehow. I did it, and received an error
> 
> error: The argument 'max_genesis_archive_unpacked_size' wasn't found

So revisiting what we do elsewhere, I changed my mind and I don't think we want this as an arg of the `program` subcommand. But, can you do the same thing with declaring the arg once, and then passing a clone (or hopefully reference) to each subcommand?

Additionally, can you please remove the duplicate constant and instead reference `solana_runtime::hardened_unpack::MAX_GENESIS_ARCHIVE_UNPACKED_SIZE` ?

Reply:
Ahh darn, yeah, the location I referenced to is in `main()` so not quite the same. Ahh ok, I have a larger refactor in mind that would address the issue, but that is definitely out of scope for this PR.

Let's leave it as is and I'm marking this conversation as resolved.